src/MagnumPlugins/BasisImageConverter/Test/BasisImageConverterTest.cpp

```diff
@@ -1425,6 +1429,58 @@ void BasisImageConverterTest::swizzle() {
         TestSuite::Compare::around(Vector4i{2}));
 }
 
+void BasisImageConverterTest::openCL() {
+    if(_manager.loadState("PngImporter") == PluginManager::LoadState::NotFound)
+        CORRADE_SKIP("PngImporter plugin not found, cannot test contents");
+
+    Containers::Pointer<AbstractImporter> pngImporter = _manager.instantiate("PngImporter");
+    CORRADE_VERIFY(pngImporter->openFile(Utility::Path::join(BASISIMPORTER_TEST_DIR, "rgba-31x13.png")));
+    Containers::Optional<Trade::ImageData2D> original = pngImporter->image2D(0);
+    CORRADE_VERIFY(original);
+
+    Containers::Pointer<AbstractImageConverter> converter = _converterManager.instantiate("BasisImageConverter");
+    /* Default is off */
+    CORRADE_COMPARE(converter->configuration().value<bool>("use_opencl"), false);
+    converter->configuration().setValue("use_opencl", true);
+
+    std::ostringstream out;
+    Containers::Optional<Containers::Array<char>> compressedData;
+    {
+        Warning redirectWarning{&out};
+        compressedData = converter->convertToData(*original);
+    }
+
+    /* If built without OpenCL, converting falls back to CPU and still succeeds */
+    #ifndef OpenCL_FOUND
+    CORRADE_COMPARE(out.str(), "Trade::BasisImageConverter::convertToData(): OpenCL not supported, falling back to CPU encoding\n");
+    #else
+    CORRADE_COMPARE(out.str(), "");
+    #endif
```

Comment:
For easier verification of which CI jobs run with OpenCL and which not, I'd do

```suggestion
    #ifndef OpenCL_FOUND
    CORRADE_WARN("OpenCL is not available.");
    CORRADE_COMPARE(out.str(), "Trade::BasisImageConverter::convertToData(): OpenCL not supported, falling back to CPU encoding\n");
    #else
    CORRADE_INFO("OpenCL is available.");
    CORRADE_COMPARE(out.str(), "");
    #endif
```